src/widgetastic/widget/table.py

```diff
@@ -987,8 +998,10 @@ def _process_table(self):
             node = queue.popleft()
             # visit node
             children = self.browser.elements('./*[descendant-or-self::node()]', parent=node.obj)
-            for position, child in enumerate(children):
+            position = 0
+            for child in children:
```

Comment:
please make a private generator function that excludes the children so enumerate can be used again